### PR TITLE
fix: ito image payload

### PR DESCRIPTION
**Pull request by zhouhanseng**
closes #

### CODE REVIEW

File: packages/maskbook/src/plugins/ITO/SNSAdaptor/index.tsx [hunk 1]
```diff
@@ -34,12 +46,28 @@ const sns: Plugin.SNSAdaptor.Definition = {
...
         : payload.message.split(MSG_DELIMITER)[0] ?? formatEthereumAddress(payload.seller.address, 4)
-    return `A ITO with ${formatBalance(payload.total, payload.token?.decimals)} $${
-        payload.token?.symbol ?? payload.token?.name ?? 'Token'
-    } from ${sellerName}`
+    return loadingToken ? null : (
+        <div className={classes.root}>
+            <ItoLabelIcon size={14} />
+            <span className={classes.span}>{`A ITO with ${formatBalance(payload.total, tokenDetailed?.decimals)} $${
+                tokenDetailed?.symbol ?? tokenDetailed?.name ?? 'Token'
+            } from ${sellerName}`}</span>
```
Comment (septs) [inline packages/maskbook/src/plugins/ITO/SNSAdaptor/index.tsx:68]: extract this template string as variable.

Comment (septs) [reply packages/maskbook/src/plugins/ITO/SNSAdaptor/index.tsx:68]:
```suggestion
            <span className={classes.span}>{`A ITO with ${balance} $${symbol} from ${sellerName}`}</span>
```

File: packages/maskbook/src/plugins/ITO/SNSAdaptor/ITO.tsx [hunk 2]
```diff
@@ -392,7 +394,7 @@ export function ITO(props: ITO_Props) {
             summary += ' ' + formatBalance(total_remaining, token.decimals) + ' ' + token.symbol
         }
         availability?.exchange_addrs.forEach((addr, i) => {
-            const token = exchange_tokens.find((t) => t.address.toLowerCase() === addr.toLowerCase())
+            const token = exchange_tokens.find((t) => isSameAddress(t.address, addr))
```
Comment (septs) [inline packages/maskbook/src/plugins/ITO/SNSAdaptor/ITO.tsx:397]:
```suggestion
            const token = exchange_tokens.find(currySameAddress(addr))
```

**Review by septs — APPROVED** [review]

**Review by nuanyang233 — APPROVED** [review]

File: packages/maskbook/src/plugins/ITO/SNSAdaptor/CompositionDialog.tsx [hunk 3]
```diff
@@ -148,7 +168,7 @@ export function CompositionDialog(props: CompositionDialogProps) {
         tabs: [
             {
                 label: t('plugin_ito_create_new'),
-                children: usePortalShadowRoot((container) => (
+                children: usePortalShadowRoot(() => (
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/ITO/SNSAdaptor/CompositionDialog.tsx:171]: If you dont use the container, you don't need to call `usePortalShadowRoot`

**Pull request merged**